### PR TITLE
Add instructions to enable remote cluster gateways using Helm

### PR DESCRIPTION
**Pull request by johnma14**
Fixes: #9397

- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/setup/install/external-controlplane/test.sh [hunk 1]
```diff
@@ -101,19 +101,21 @@ _verify_like snip_deploy_a_sample_application_3 "$snip_deploy_a_sample_applicati
...
 _verify_contains snip_deploy_a_sample_application_4 "Hello version: v1"
 
-echo y | snip_enable_gateways_1
-#echo y | snip_enable_gateways_2
-
-_verify_like snip_enable_gateways_3 "$snip_enable_gateways_3_out"
+#echo y | snip_enable_gateways_1
+echo y | snip_enable_gateways_2
 
 snip_enable_gateways_4
```
[the remark this answers is not in the document]
Comment (johnma14) [reply content/en/docs/setup/install/external-controlplane/test.sh:112]: Done.

**Comment by johnma14** [comment]
Made recommended updates and ready for review. Thanks.

File: content/en/docs/setup/install/external-controlplane/index.md [hunk 2]
```diff
@@ -424,6 +424,9 @@ including gateways, if needed.
 
 1. Enable an ingress gateway on the remote cluster:
```
Comment (ericvn) [inline content/en/docs/setup/install/external-controlplane/index.md:425]: The numbering isn't quite correct. We have 3 number 1's at the top of the list. I'm not sure if indentation of the tabsets will make things correct or not.

Comment (johnma14) [reply content/en/docs/setup/install/external-controlplane/index.md:425]: @ericvn That worked. Made the changes and it looks good. Thanks for the help. Appreciate it.

**Review by ericvn — APPROVED** [review]

**Comment by craigbox** [comment]
Do we want this in the current (1.11) docs?

**Pull request merged**